README.md

```diff
@@ -1,2 +1,307 @@
+
+The command specialized for gOTU analysis, **gotu**, is a simplified wrapper of the full-power command, **classify**, which has more parameters to allow for extended features and controls. We will start with this simple analysis before diving into the complex and flexible usages.
+
+### Background
+
+The notion of “gOTU” (pronounced as "go-to") is the minimal unit for community ecology studies based on shotgun metagenome or other forms of whole-genome microbiome data. It is in constrast to conventional practices, in which taxonomic units such as genera or species were used. Therefore,  gOTU is analogous to sOTU in 16S rRNA studies. The advantage of using gOTU includes 1) highest-possible resolution, 2) independent from taxonomy which is coarse and error-prone as a classification system. 3) allowing for phylogeny-based analysis such as Faith’s PD and UniFrac. The last part is enhanced by the “web of life” reference phylogeny.
+
+### gOTU table generation
+
+To generate a gOTU table, one needs a multiplexed alignment file, or a directory of per-sample alignment files. These files can be generated by aligning sequencing data against a reference genome database. We recommend using [**SHOGUN**](https://github.com/knights-lab/SHOGUN) with the "Web of Life" database (**WoL**, available for download at: https://biocore.github.io/wol/). For example:
+
+```bash
+shogun align -a bowtie2 -d WoLr1 -i input.fasta -o .
+```
+
+Then one can run Woltka to convert the alignment file(s) into a gOTU table:
+
+```bash
+woltka gotu -i alignment.bowtie2.sam -o table.tsv
+```
+
+The output file `table.tsv` is a table with rows as genome IDs (gOTUs), columns as sample IDs, and cell values as counts of gOTUs in samples.
+
+Note: Both SHOGUN and WoL are available at the [**Qiita**](https://qiita.ucsd.edu/) server. If you are a Qiita user, the alignment file can be automatically generated and downloaded from the Qiita interface.
+
+### gOTU analysis using QIIME 2
+
+With QIIME, the gOTU table can be converted into BIOM format:
+
+```bash
+biom convert --to-hdf5 -i table.tsv -o table.biom
+```
+
+The BIOM table can be imported into a QIIME artifact:
+
+```bash
+qiime tools import --type FeatureTable[Frequency] --input-path table.biom --output-path table.qza
+```
+
+These intermediate steps are automated if you use the [QIIME 2 plugin of Woltka](woltka/q2).
+
+One can then investigate the microbiome by applying classical QIIME analyses on the gOTU table. For example, with the [WoL reference phylogeny](https://biocore.github.io/wol/), one can do:
+
+```bash
+qiime diversity core-metrics-phylogenetic \
+  --i-phylogeny tree.qza \
+  --i-table table.qza \
+  --p-sampling-depth 1000 \
+  --m-metadata-file metadata.tsv \
+  --output-dir .
+```
+
+### Alignment ambiguity
+
+It is quite common that one query sequence can be aligned to multiple reference genomes. In such cases, Woltka by default counts each gOTU as 1 / _k_, where _k_ is the total number of matching genomes.
```

Comment:
Outside of the scope of this PR, but I wonder if it would be effective to do something like bracken in these instances? If this is interesting to you, can you open an issue and it can be addressed in the future?

Reply:
I totally agree with your suggestion. I envision that there will be an algorithm, which combines LCA, majority-rule, and a bracken-like function (but based on sequence alignment instead of k-mer signature). Issue created (#5)